packages/vite/src/node/tsconfig.json

```diff
@@ -5,9 +5,9 @@
   "compilerOptions": {
     // compile away optional-chaining-operator
     // node support table: https://node.green/#ES2020-features-optional-chaining-operator-----
```

Comment:
```suggestion
```
I think this comment should be removed since we no longer transpile optional chaining operator.